x/ibc/03-connection/types/version.go

```diff
@@ -1,96 +1,171 @@
 package types
 
-// GetCompatibleVersions returns an ordered set of compatible IBC versions for the
-// caller chain's connection end.
+import (
+	"fmt"
+	"strings"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	host "github.com/cosmos/cosmos-sdk/x/ibc/24-host"
+)
+
+var (
+	// DefaultIBCVersion represents the latest supported version of IBC.
+	// The current version supports only ORDERED and UNORDERED channels.
+	DefaultIBCVersion = CreateVersionString("1", []string{"ORDERED channel", "UNORDERED channel"})
```

Comment:
Think this is a little cleaner

```suggestion
	DefaultIBCVersion = CreateVersionString("1", []string{"ORDERED", "UNORDERED"})
```

Reply:
shouldn't the version be 1.0.0 too?

Reply:
it can be, this is still a decision yet to be made from my understanding

Reply:
also, will commit the suggestion, but this default version is just a placeholder for what is eventually decided. I think it could worth considering requiring either ordered or unordered channels to be implemented as a core feature in version 1 of IBC